src/components/header_menu.jsx

```diff
@@ -14,23 +14,23 @@ import { getDriveLink } from 'lib/utils'
 import { NotePath } from './notes/List/NotePath'
 import { WithBreakpoints } from './notes/List/WithBreakpoints'
 import styles from './header_menu.styl'
+import useFileWithPath from 'hooks/useFileWithPath'
 
 const HeaderMenu = ({
   homeHref,
   backFromEditing,
   editorCorner,
   isPublic,
   file,
-  client,
+  cozyClient,
```

Comment:
to me you should not pass the client in props. You should use `useClient()` inside the component to get it

Reply:
updated